Jint/Native/Function/ScriptFunctionInstance.cs

```diff
@@ -94,6 +95,16 @@ public override JsValue Call(JsValue thisArg, JsValue[] arguments)
 
                     if (result.Type == Completion.Throw)
                     {
+                        if (result.Value.HasValue && result.Value.Value.IsObject())
+                        {
+                            var e = result.Value.Value.ToObject() as Exception;
+                            if (e != null)
+                            {
+                                e.Rethrow();
+                                throw new InvalidOperationException("Clr is not working");
```

Comment:
Ah, now I get the joke! Well, please remove the line then, even more since apparently you just need to use `throw;` and it will display a warning otherwise.